lib/react_client/react_interop.dart

```diff
@@ -53,6 +53,7 @@ abstract class React {
   external static Function useCallback(Function callback, List dependencies);
   external static ReactContext useContext(ReactContext context);
   external static JsRef useRef([dynamic initialValue]);
+  external static void useImperativeHandle(JsRef ref, Function() createHandle, [List dependencies]);
```

Comment:
Based on whether or not my typing information of `hooks.useImperativeHandle` is correct, this would be updated to match:
```suggestion
  external static void useImperativeHandle(JsRef ref, void Function() createHandle, [List<dynamic> dependencies]);
```